README.md

```diff
@@ -61,6 +61,51 @@ You can also always find the full command-line options that are available with
 $ cargo fuzz --help
 ```
 
+## Generating code coverage information
```

Comment:
This documentation is probably better suited for [the book](https://github.com/rust-fuzz/book)?

Reply:
I completely agree; however, this would require starting a coverage-report-generation section in the book and it might take some time to do. I suggest to keep it here until one of us finds the time to do that?

Reply:
FWIW, I'm fine having this here for now.